themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -1042,6 +1042,31 @@ table-of-password-policies=Table of Password Policies
 add-policy.placeholder=Add policy...
 policy-type=Policy Type
 policy-value=Policy Value
+webauthn-policy=WebAuthn Policy
+webauthn-rp-entity-name=Relying Party Entity Name
+webauthn-rp-entity-name.tooltip=Human-readable server name as WebAuthn Relying Party
+webauthn-signature-algorithms=Signature Algorithms
+webauthn-signature-algorithms.tooltip=What signature algorithms should be used for Authentication Assertion.
+webauthn-rp-id=Relying Party ID
+webauthn-rp-id.tooltip=This is ID as WebAuthn Relying Party. It must be origin's effective domain.
+webauthn-attestation-conveyance-preference=Attestation Conveyance Preference
+webauthn-attestation-conveyance-preference.tooltip=It tells an authenticator the preference of how to generate an attestation statement.
+webauthn-authenticator-attachment=Authenticator Attachment
+webauthn-authenticator-attachment.tooltip=It tells an authenticator an acceptable attachment pattern.
+webauthn-require-resident-key=Require Resident Key
+webauthn-require-resident-key.tooltip=It tells an authenticator create a public key credential as Resident Key or not.
+webauthn-user-verification-requirement=User Verification Requirement
+webauthn-user-verification-requirement.tooltip=It tells an authenticator confirm actually verifying a user.
+webauthn-create-timeout=Timeout
+webauthn-create-timeout.tooltip=Timeout value for creating user's public key credential
```

Comment:
Accouding to [WebAuthn API specification](https://www.w3.org/TR/webauthn/#dom-publickeycredentialcreationoptions-timeout), this option is in milliseconds.

However, people are unfamiliar with milliseconds, keycloak and its UI treats it in seconds and convert it when calling navigator.credentials.create().

We can call navigator.credentials.create() without this timeout option because this option is not mandatory according to [WebAuthn API specification](https://www.w3.org/TR/webauthn/#dom-publickeycredentialcreationoptions-timeout). Therefore, I've decided that if this timeout value is 0, keycloak calls navigator.credentials.create() without this timeout option. 

What do you think about the following?

```text
webauthn-create-timeout.tooltip=Timeout value for creating user's public key credential in seconds. If set to 0, this timeout option is not adapted.
```

Reply:
+1 for the tooltip proposal you mentioned.